### PR TITLE
Fixed #583 Navigation module highlighting added

### LINKED ISSUE #583: Navigation module highlighting
SuiteCRM currently does not highlight the module you're currently using in the navigation.

bildschirmfoto 2015-10-20 um 12 33 47

improved usability:
bildschirmfoto 2015-10-20 um 12 37 05

SuiteCRM Version: 7.3.1

### PR DESCRIPTION
**Pull request by shogunpol**
Fixed #583 Navigation module highlighting added
## Description

Added highlight currently used module

## Motivation and Context

Improve user experience
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: themes/SuiteR/js/navhighlight.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
+    //adding the css classes to the current-module-nav-item & link
```
Comment (daniel-samson) [inline themes/SuiteR/js/navhighlight.js]: A better solution would be to add the classes into the _headerModuleList.tpl

File: themes/SuiteR/tpls/_head.tpl [hunk 2]
```diff
@@ -73,4 +73,7 @@
     <link rel="stylesheet" type="text/css" href="themes/SuiteR/css/colourSelector.php">
     <script type="text/javascript" src='{sugar_getjspath file="themes/SuiteR/js/jscolor.js"}'></script>
     <script type="text/javascript" src='{sugar_getjspath file="cache/include/javascript/sugar_field_grp.js"}'></script>
+    <!-- include navigation highlighting -->
+    <link rel="stylesheet" type="text/css" href="themes/SuiteR/css/navhighlight.css">
```
Comment (daniel-samson) [inline themes/SuiteR/tpls/_head.tpl]: see above

File: themes/SuiteR/css/navhighlight.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.current-selected-module {
```
Comment (daniel-samson) [inline themes/SuiteR/css/navhighlight.css]: this should be included in the style.css file

File: themes/SuiteR/tpls/_headerModuleList.tpl [hunk 4]
```diff
@@ -82,6 +82,9 @@
                         {foreach from=$GCLS item=GCL name=gcl key=gcl_key}
                             <li role="presentation">
                                 <a id="{$gcl_key}_link" href="{$GCL.URL}"{if !empty($GCL.ONCLICK)} onclick="{$GCL.ONCLICK}"{/if}>{$GCL.LABEL}</a>
+                                <!-- include navigation highlighting -->
+                                <link rel="stylesheet" type="text/css" href="themes/SuiteR/css/navhighlight.css">
```
Comment (daniel-samson) [inline themes/SuiteR/tpls/_headerModuleList.tpl]: see above

Comment (gymad) [inline themes/SuiteR/tpls/_headerModuleList.tpl]: Can we use less theme-name specific templates here?

File: themes/SuiteR/tpls/_head.tpl (hunk 2 above)
Comment (gymad) [inline themes/SuiteR/tpls/_head.tpl]: Can we use less theme-name specific templates here?

File: themes/SuiteR/js/navhighlight.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+    $("li.topnav:has(span.currentTab)").addClass("current-selected-module");
```
Comment (gymad) [inline themes/SuiteR/js/navhighlight.js]: add class in .tpl files if it's possible

File: themes/SuiteR/css/navhighlight.css [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+    background-color: #5880BA;
+}
+/* optional: reimplement the link underline to support the visibility uncomment lines below */
+
+/*.current-selected-module-link {*/
```
Comment (gymad) [inline themes/SuiteR/css/navhighlight.css]: remove it instead comment out

**Pull request merged**